PWGDQ/Core/VarManager.h

```diff
@@ -787,6 +791,7 @@ void VarManager::FillTrack(T const& track, float* values)
     values[kTrackCSnpSnp] = track.cSnpSnp();
     values[kTrackCTglTgl] = track.cTglTgl();
     values[kTrackC1Pt21Pt2] = track.c1Pt21Pt2();
+    values[kPt_in] = values[kPin] / std::sqrt(1.0 + track.tgl() * track.tgl());
```

Comment:
Since this variable require some computation, please add an "if(fgUsedVars[kPt_in])" around it.
But first we need to understand why is this variable useful.